src/main/scala/com/gu/octopusthrift/services/PayloadValidator.scala

```diff
@@ -8,17 +8,23 @@ import scala.util.{ Success, Try }
 
 object PayloadValidator extends Logging with CustomMetrics with DeadLetterQueue {
```

Comment:
I'm having trouble running my tests locally because of these traits `CustomMetrics ` and `DeadLetterQueue`. If we are not using them anymore, can they be removed?